hezar/embeddings/embedding.py

```diff
@@ -26,19 +25,20 @@
 
 # Check if the right combo of gensim/numpy versions are installed
 def _verify_gensim_installation():
-    if (
```

Comment:
@HamedBabaei It's better to keep this unchanged as it is for better readability.

Reply:
@arxyzan I see, this is done now!